services/cd-service/pkg/valid/validations.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	MaxAppNameLen  = 39
+	MaxAppNameLen  = 70
```

Comment:
We have to double check this.
The app name is also sent to argoCd via CRD and I vaguely remember we did this because of a technical limitations there. (maybe dns)

Reply:
[This](https://github.com/argoproj/argo-cd/issues/5595) specifies that the limit for app names is 63 characters.

Reply:
Yes, you can put 70 here but then argocd breaks.

Reply:
@ahmed-nour-fdc exactly - also see my comment to the PR. I tested it in my own instance.

Reply:
okay, i'll adapt it next week. Sorry for this torn apart review

Reply:
@sven-urbanski-freiheit-com I updated it to use `KUBERPULT_MAX_APP_NAME_LENGTH` env variable.
As this is used inside the `validation.go` I had to somehow read the env there.
If you can think of a better approach - go ahead and sketch it and I'll implement it :)